packages/ember-htmlbars/lib/node-managers/view-node-manager.js

```diff
@@ -122,6 +126,10 @@ ViewNodeManager.prototype.rerender = function(env, attrs, visitor) {
       env.renderer.willUpdate(component, snapshot);
 
       if (component._renderNode.shouldReceiveAttrs) {
+        if (component.propagateAttrsToThis) {
```

Comment:
Fixed to use underscore prefix in https://github.com/emberjs/ember.js/commit/6c5f87d61d1893654a1f28f0b25902c1c782311b